### ISSUE TITLE
Accessibility Issue: Checkbox input elements in List Item component

### ISSUE THREAD

**Issue by jennaly** [the-collab-lab/tcl-53-smart-shopping-list#50]
## Description
Input element (checkbox) in List Item component currently has the property ```display: none```.  Keyboard users aren't able to focus on the element when tabbing through the page and therefore cannot "check" an item with the enter key. 

## Acceptance Criteria
- [ ] Use CSS to style the checkbox input element directly
- [ ] Remove FontAwesomeIcon SVG circle and checked circle icons

**Comment by jennaly** [comment]
closing issue because checkboxes can be "checked" with space key
